src/Controller/Admin/SecurityController.php

```diff
@@ -64,13 +64,26 @@ public function newAction(Request $request, PasswordResetService $passwordReset,
 
         if ($form->isSubmitted() && $form->isValid()) {
             $em = $this->getDoctrine()->getManager();
+            $repository = $em->getRepository(LocalAccount::class);
 
+            $email = $form["email"]->getData();
+            $sameMail = $repository->findBy(['email' => $email]);
+
+            if (count($sameMail) != 0) {
```

Comment:
This code checks whether an e-mail already exist, and therefore "validates" the user input. To do this, we use validation constraints instead. Please review the [UniqueEntity](https://symfony.com/doc/current/reference/constraints/UniqueEntity.html) constraint to see how to implement this